action.yml

```
@@ -25,6 +25,9 @@ inputs:
     default: 'Stale'
   exempt-pr-label:
     description: 'The label to apply when a pull request is exempt from being marked stale'
+  only-labels:
+    description: 'The labels to apply when only specific issues/pull requests to mark as stale'
```

I'd recommend changing the description to better describe the purpose of the input. Something like:

```suggestion
    description: 'Comma separated list of labels an issue/pull request needs to have'
```

---

Also mention it can be comma-separated list of labels